data/exclusions-disease-gene.tsv

```diff
@@ -1,8 +1,9 @@
 omim_id	mondo_id	mondo_label	orcid	exclusion_reason_comment
-OMIM:603956	MONDO:0002974	cervical cancer'	https://orcid.org/0000-0002-4142-7153	evidence of various genes involved
-OMIM:619151	MONDO:0030894	"AMED syndrome, digenic'"	https://orcid.org/0000-0002-4142-7153	digenic
+OMIM:603956	MONDO:0002974	cervical cancer	https://orcid.org/0000-0002-4142-7153	evidence of various genes involved
+OMIM:619151	MONDO:0030894	AMED syndrome, digenic	https://orcid.org/0000-0002-4142-7153	digenic
 OMIM:158901	MONDO:0008031		https://orcid.org/0000-0002-4142-7153	digenic
-OMIM:108770	MONDO:0007171	atrial standstill 1'	https://orcid.org/0000-0002-4142-7153	digenic
-OMIM:620040	MONDO:0031057	"dyskeratosis congenita, digenic'"	https://orcid.org/0000-0002-4142-7153	digenic
-OMIM:619478	MONDO:0030355	"facioscapulohumeral muscular dystrophy 4, digenic'"	https://orcid.org/0000-0002-4142-7153	digenic
-OMIM:300818	MONDO:0010438	paroxysmal nocturnal hemoglobinuria 1	https://orcid.org/0000-0002-4142-7153	"disease caused by a somatic mutation, therefore a gene association stating this is due to a germline mutation should not be added"
\ No newline at end of file
+OMIM:108770	MONDO:0007171	atrial standstill 1	https://orcid.org/0000-0002-4142-7153	digenic
+OMIM:620040	MONDO:0031057	dyskeratosis congenita, digenic	https://orcid.org/0000-0002-4142-7153	digenic
+OMIM:619478	MONDO:0030355	facioscapulohumeral muscular dystrophy 4, digenic	https://orcid.org/0000-0002-4142-7153	digenic
+OMIM:300818	MONDO:0010438	paroxysmal nocturnal hemoglobinuria 1	https://orcid.org/0000-0002-4142-7153	"disease caused by a somatic mutation, therefore a gene association stating this is due to a germline mutation should not be added"
+OMIM:278850	MONDO:0010218	46,XX sex reversal 2	https://orcid.org/0000-0002-4142-7153	"evidence that familial 46,XX sex reversal is caused by heterozygous duplication or triplication of a 68-kb regulatory region (XXSR) -584 to -516 kb upstream of the SOX9 gene (608160) on chromosome 17q24."
```

Comment:
With the new definition that the exclusion file is for gene associations that are otherwise valid by the "rule", but after curator review it is decided that the gene association should not be added into Mondo as disease defining only `OMIM:278850	MONDO:0010218	46,XX sex reversal 2` should be maintained in the exclusion file. See comment [here](https://github.com/monarch-initiative/mondo/pull/8624/files#r1936458946).

Reply:
OK. There was a lot of rows changed as well, but it looks like it may have just been TSV formatting. I thought Sabrina had manually removed/added other entries. I thought that the one case in that comment was just 1 example; not the only row that should be in here.

If that is indeed the only entry that should be in here, I'll change this file now. I'll do it in this PR, as well as #188 (and run the release again for it).